packages/spacecat-shared-data-access/test/it/db.test.js

```diff
@@ -99,13 +99,17 @@ const TEST_DA_CONFIG = {
   tableNameConfigurations: 'spacecat-services-configurations',
   tableNameSiteTopPages: 'spacecat-services-site-top-pages',
   tableNameExperiments: 'spacecat-services-experiments',
+  tableNameApiKeys: 'spacecat-services-api-keys',
+  tableNameImportJobs: 'spacecat-services-import-jobs',
+  tableNameImportUrls: 'spacecat-services-import-urls',
   indexNameAllSites: 'spacecat-services-all-sites',
   indexNameAllKeyEventsBySiteId: 'spacecat-services-key-events-by-site-id',
   indexNameAllSitesOrganizations: 'spacecat-services-all-sites-organizations',
   indexNameAllOrganizations: 'spacecat-services-all-organizations',
   indexNameAllOrganizationsByImsOrgId: 'spacecat-services-all-organizations-by-ims-org-id',
   indexNameAllSitesByDeliveryType: 'spacecat-services-all-sites-by-delivery-type',
   indexNameAllLatestAuditScores: 'spacecat-services-all-latest-audit-scores',
+  indexNameAllImportJobsByStatus: 'spacecat-services-all-import-jobs-by-status',
```

Comment:
while it is still nice to have these; I don't think any of these changes has an actual impact for the IT tests since 1) we have fallback table names in data-access; 2) import table in the IT tests is already very tiny